lib/model.js

```diff
@@ -1000,16 +1000,18 @@ Connection.from = function(url) {
  * Helper function to improve the Atlas user experience by
  * providing better default values.
  *
- * @param {String} url  The connection string URL.
- * @param {String} mongodb_password  The
+ * @param {String} url - The connection string URL.
+ * @param {String} mongodb_password - The mongodb_password
+ *   which the user may need to change.
  * @returns {Object} Connection attributes to override
  * @private
  */
 Connection._improveAtlasDefaults = function(url, mongodb_password) {
   var atlasConnectionAttrs = {};
   if (Connection.isAtlas(url)) {
-    atlasConnectionAttrs.ssl = 'UNVALIDATED';
-    if (mongodb_password === 'PASSWORD') {
+    atlasConnectionAttrs.ssl = 'SYSTEMCA';
+    if (mongodb_password.length <= 20 &&
```

Comment:
👍 https://en.wikipedia.org/wiki/Zero_one_infinity_rule and https://github.com/mongodb-js/connection-model/commit/fa4f56a89478928819e612b1f04a8845df73d104